keras/backend/theano_backend.py

```diff
@@ -202,6 +202,12 @@ def any(x, axis=None, keepdims=False):
     return T.any(x, axis=axis, keepdims=keepdims)
 
 
+def all(x, axis=None, keepdims=False):
```

Comment:
It would be useful to add this `all` op to the backend tests: https://github.com/fchollet/keras/blob/master/tests/keras/backend/test_backends.py#L152